config/testgrids/config.yaml

```diff
@@ -251,6 +251,12 @@ dashboards:
   dashboard_tab:
   - name: cel-go
     test_group_name: cel-go-conformance-tests
+    file_bug_template: # The URL template to visit when filing a bug
```

Comment:
Since the test group is defined in `conformance/conformance-all.yaml`, please put the google-cel dashboard into that file. It's the only row in [that dashboard](https://testgrid.k8s.io/google-cel#Summary)

Reply:
Done